cmd/cstor-pool-mgmt/volumereplica/volumereplica.go

```diff
@@ -189,6 +196,44 @@ func builldVolumeCloneCommand(cStorVolumeReplica *apis.CStorVolumeReplica, snapN
 	return cloneVolCmd
 }
 
+// CreateVolumeBackup sends cStor snapshots to remote location specified by backupcstor.
+func CreateVolumeBackup(bkp *apis.BackupCStor) error {
+	var cmd []string
+	var retryCount int
+	var err error
+
+	// Parse capacity unit on CVR to support backward compatibility
+	cmd = builldVolumeBackupCommand(bkp.ObjectMeta.Labels["cstorpool.openebs.io/uid"], bkp.Spec.VolumeName, bkp.Spec.PrevSnapName, bkp.Spec.SnapName, bkp.Spec.BackupDest)
+
+	glog.Infof("Backup Command for volume: %v created, Cmd: %v\n", bkp.Spec.VolumeName, cmd)
+
+	for retryCount < MaxBackupRetryCount {
+		stdoutStderr, err := RunnerVar.RunCombinedOutput("/usr/local/bin/execute.sh", cmd...)
+		if err != nil {
+			glog.Errorf("Unable to start backup %s. error : %v retry:%v :%s", bkp.Spec.VolumeName, string(stdoutStderr), retryCount, err.Error())
+			retryCount++
+			time.Sleep(BackupRetryDelay * time.Second)
+			continue
+		}
+		break
+	}
+	return err
```

Comment:
This err will always be nil. 
At line 211 we can remove shorthand(:=) declaration to fix this.